scripts/hack-express-server.js

```diff
@@ -0,0 +1,21 @@
+const commander = require('commander');
```

Comment:
This file is a hack to expose the functional testing express server as a cli for heroku. I'd like to see the express server broken out into either a new toolkit package or a package other than the functional testing package since we use the express server to host on heroku.